sentry-kotlin-multiplatform/src/appleMain/kotlin/io/sentry/kotlin/multiplatform/extensions/SentryOptionsExtension.kt

```diff
@@ -0,0 +1,16 @@
+package io.sentry.kotlin.multiplatform.extensions
+
+import cocoapods.Sentry.SentryEvent
+import io.sentry.kotlin.multiplatform.CocoaSentryOptions
+import io.sentry.kotlin.multiplatform.nsexception.dropKotlinCrashEvent
+import NSException.Sentry.SentryEvent as NSExceptionSentryEvent
+
+fun io.sentry.kotlin.multiplatform.SentryOptions.toCocoaSentryOptions(): CocoaSentryOptions {
```

Comment:
```suggestion
fun SentryOptions.toCocoaSentryOptions(): CocoaSentryOptions {
```

We can import the package and use the class name directly

Reply:
Oh thanks I didn't even notice that